site-config/anvil-catalog/dev/config.ts

```diff
@@ -16,15 +17,25 @@ import { buildNavigation } from "./layout/header/navigation/navigation";
 // Template constants
 const APP_TITLE = "AnVIL Dataset Catalog";
 const BROWSER_URL = "https://anvilproject.dev.clevercanary.com";
+const BUILD_DATE = process.env.NEXT_PUBLIC_BUILD_DATE;
 const EXPLORER_URL = "https://explore.anvilproject.dev.clevercanary.com";
+const GIT_HASH = process.env.NEXT_PUBLIC_GIT_HASH;
+const GIT_HUB_REPO_URL = "https://github.com/DataBiosphere/data-browser";
 const HOME_PAGE_PATH = ROUTES.CONSORTIA;
 const PORTAL_URL = "https://anvilproject.dev.clevercanary.com";
 const SLOGAN = "NHGRI Analysis Visualization and Informatics Lab-space";
+const VERSION = process.env.NEXT_PUBLIC_VERSION;
 
 export function makeConfig(
   browserUrl: string,
   portalUrl: string,
-  explorerUrl: string
+  explorerUrl: string,
+  gitHubUrl: string = GIT_HUB_REPO_URL,
```

Comment:
Hi @frano-m, can you do this without modifying the makeConfig? The provider or component can directly access the environment.

Reply:
Sure.

Reply:
Oh I misread this... it's the repo URL.